src/components/avatar-image.tsx

```diff
@@ -1,13 +1,14 @@
+import Image from "next/image";
 import React from "react";
 import styled from "styled-components";
 
-const defaultAvatarImg =
-  "https://gravatar.com/avatar/6ae852fa3a8b1c79dba3f7dc883c1760?s=200&d=mp&r=x";
+import defaultAvatarImg from "/public/icons/avatar_default.svg";
 
-const StyledImg = styled.img`
-  width: 28px;
-  height: 28px;
+const StyledImg = styled(Image)`
```

Comment:
```suggestion
const StyledImg = styled.image`
```

This will allow us to remove  `images: { domains: ["lh3.googleusercontent.com"] }`. I guess it’s fair to assume that google avatars are optimised already, so we don’t need to pass them through the Next.js runtime optimizer. Not sure how it works on Netlify BTW – that’s another source of complexity.

Also `"lh3.googleusercontent.com"` might not be the only source of avatars. Googling for other `"lhN.googleusercontent.com"` does show up stuff, so `lh3` might be some CDN zone etc.

Reply:
I'm not sure the variant with IMG still have NextJS optimization.

Reply:
Plain `img` tags don’t benefit from Next.js optimisations – that’s correct. The net benefit of this optimisation at this point can be very small or even negative for this project. This does not mean that we won’t want to return to the topic later!

AFAIU, external avatars are optimised already, so they don’t need to be shown inside `<Image />` either way.